build/package/circleci/orb.yml

```diff
@@ -1,65 +1,77 @@
 version: 2.1
-description: Job for finding and sending feature flag code references to LaunchDarkly
+description: "Job for finding and sending feature flag code references to LaunchDarkly. Code references documentation: https://docs.launchdarkly.com/v2.0/docs/git-code-references"
 
-executors:
-  default:
-    parameters:
-      proj_key:
-        type: string
-      base_uri:
-        type: string
-      context_lines:
-        type: integer
-      default_branch:
-        type: string
-      exclude:
-        type: string
-      repo_type:
-        type: string
-      repo_url:
-        type: string
-      commit_url_template:
-        type: string
-      hunk_url_template:
-        type: string
-    environment:
-      LD_PROJ_KEY: << parameters.proj_key >>
-      LD_BASE_URI: << parameters.base_uri >>
-      LD_EXCLUDE: << parameters.exclude >>
-      LD_CONTEXT_LINES: << parameters.context_lines >>
-      LD_REPO_TYPE: << parameters.repo_type >>
-      LD_REPO_URL: << parameters.repo_url >>
-      LD_COMMIT_URL_TEMPLATE: << parameters.commit_url_template >>
-      LD_HUNK_URL_TEMPLATE: << parameters.hunk_url_template >>
-      LD_DEFAULT_BRANCH: << parameters.default_branch >>
-    docker:
-      - image: launchdarkly/ld-find-code-refs:0.5.0
-commands:
-  find-flags:
-    steps:
-      - checkout:
-          path: /repo
-      - run:
-          name: Find flag references
-          command: |
-            ld-find-code-refs \
-              -accessToken=${LD_ACCESS_TOKEN} \
-              -projKey=${LD_PROJ_KEY} \
-              -exclude=${LD_EXCLUDE} \
-              -contextLines=${LD_CONTEXT_LINES} \
-              -baseUri=${LD_BASE_URI} \
-              -repoType=${LD_REPO_TYPE} \
-              -repoUrl=${LD_REPO_URL} \
-              -repoName=${CIRCLE_PROJECT_REPONAME} \
-              -updateSequenceId=${CIRCLE_BUILD_NUM} \
-              -defaultBranch=${LD_DEFAULT_BRANCH} \
-              -commitUrlTemplate=${LD_COMMIT_URL_TEMPLATE} \
-              -hunkUrlTemplate=${LD_HUNK_URL_TEMPLATE} \
-              -dir=/repo
+examples:
+  minimal_config:
+    description: Minimal configuration
+    usage:
+      version: 2.1
+      orbs:
+        launchdarkly: launchdarkly/ld-find-code-refs@0.5.0
+      workflows:
+        main:
+          jobs:
+            - launchdarkly/find-code-references:
+                access_token: "${LD_ACCESS_TOKEN}"
+                proj_key: 'YOUR_LAUNCHDARKLY_PROJECT_KEY'
+                debug: true
```

Comment:
out of curiosity, why set debug:true in a minimal config

Reply:
Given that debug logs help so much with potential configuration errors, I think having it turned on in examples will help nudge users to turn it on